addon-resizer/go.mod

```diff
@@ -1,35 +1,42 @@
 module k8s.io/autoscaler/addon-resizer
 
-go 1.16
+go 1.17
```

Comment:
Why not go 1.19? It sis still getting security updates while 1.17 isn't

Reply:
Done